packages/@aws-cdk/aws-redshift-alpha/test/database-query-provider/privileges.test.ts

```diff
@@ -162,7 +182,46 @@ describe('update', () => {
       Sql: `REVOKE INSERT, SELECT ON ${tableName} FROM ${username}`,
     }));
     expect(mockExecuteStatement).toHaveBeenCalledWith(expect.objectContaining({
-      Sql: `GRANT DROP ON ${newTableName} TO ${username}`,
+      Sql: `GRANT DROP ON ${tableName} TO ${username}`,
     }));
   });
-});
+
+  test('does not replace when table id is changed', async () => {
```

Comment:
shouldn't we want replacement when table id changes?

Reply:
When the table id is changed, the privilege will change to append a new one onto the new table. However the `UserTablePrivilege` construct will not need replacing. Therefore, this test description makes sense from that perspective, which is the notation the other tests are using.